Indicators/PythonIndicator.cs

```diff
@@ -140,8 +140,8 @@ protected override decimal ComputeNextValue(IBaseData input)
         {
             using (Py.GIL())
             {
-                _isReady = _indicator.Update(input) ?? _indicator.IsReady;
-                return _indicator.Value;
+                _isReady = _indicator?.Update(input) ?? _indicator?.IsReady;
+                return _indicator?.Value;
```

Comment:
This wouldn't be required anymore, it shouldn't be null really if it is seems there's a bug?